src/parser/jobs/whm/modules/Lilies.tsx

```diff
@@ -120,6 +133,12 @@ export class Lilies extends CoreGauge {
 
 	}
 
+	private checkAfflatusAvailability() {
+		if (this.lilyGauge.value !== 0) {
```

Comment:
Should add in a check for Blood Lily Gauge so that we dont encourage overcapping